openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -352,41 +352,52 @@ const Ingestion: React.FC<IngestionProps> = ({
     });
   };
 
+  const getIngestionStatusState = (
+    current: IngestionCurrentState,
+    ingestion: IngestionPipeline,
+    text: string
+  ) => {
+    return current.id === ingestion.id ? (
+      current.state === 'success' ? (
+        <FontAwesomeIcon icon="check" />
+      ) : (
+        <Loader size="small" type="default" />
+      )
+    ) : (
+      text
+    );
+  };
+
```

Comment:
let's move it to utils and use it everywhere

Reply:
@ShaileshParmar11  Done!